packages/react-components/react-radio/src/stories/RadioGroup/RadioGroupDefault.stories.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { Label } from '@fluentui/react-label';
 import { tokens } from '@fluentui/react-theme';
 import { useId } from '@fluentui/react-utilities';
-import { Radio, RadioGroup, RadioGroupProps } from '../index';
+import { Radio, RadioGroup, RadioGroupProps } from '@fluentui/react-radio';
```

Comment:
_nit:_ You could split these up into

```suggestion
import { Radio, RadioGroup } from '@fluentui/react-radio';
import type { RadioGroupProps } from '@fluentui/react-radio';
```